packages/components/src/__actions__/Menu/v3/_docs/ApiSpecification.mdx

```diff
@@ -28,7 +28,7 @@ import * as exampleStories from "./Menu.stories"
 
 <SbContent className="mb-24">
 
-  <KAIOInstallation exportNames={["Menu", "MenuTrigger", "MenuItem"]} family="overlays" version="3" />
+  <KAIOInstallation exportNames={["Menu", "MenuTrigger", "MenuItem"]} family="actions" version="3" />
 
   <Source code={"import { Popover, Section, Header } from \"@kaizen/components/v3/react-aria-components\""} language="tsx" />
```

Comment:
Only the Popover is used, do we need to show imports for the others ones ?

Reply:
the others are used in other examples and are still relevant to menu. We just don't use them in the quick start because they are not used that often